modules/openxoutstreamBidAdapter.js

```diff
@@ -83,7 +74,7 @@ function buildCommonQueryParamsFromBids(bids, bidderRequest) {
     tws: getViewportDimensions(isInIframe),
     be: 1,
     bc: bids[0].params.bc || `${BIDDER_CONFIG}_${BIDDER_VERSION}`,
-    auid: '540141567',
+    auid: bids[0].params.unit,
```

Comment:
Why are you only looking at bidRequest[0] in this code? Sounds like there's an underlying assumption that outstream adunits will be in their own separate auctions apart from other adunits?

Reply:
@bretg I updated the adapter to make multiple requests now